f8a_worker/utils.py

```diff
@@ -632,7 +632,8 @@ def store_data_to_s3(arguments, s3, result):
         logger.error(e)
 
 
-@tenacity.retry(reraise=True, stop=tenacity.stop_after_attempt(3), wait=tenacity.wait_fixed(1))
+@tenacity.retry(stop=tenacity.stop_after_attempt(5),
+                wait=tenacity.wait_exponential(multiplier=5, min=15, max=80))
```

Comment:
whats the total time it will wait and how many retries will it do before aborting?

Reply:
It will try for 5 times and wait time will increase exponentially, it will wait for 2^x * 5 second (where x is iteration count)  between each retry starting with 15 seconds, then up to 20 seconds, then 30 seconds afterwards.

Reply:
when will it stop eventually?

Reply:
After 5 attempts